### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by FredKSchott**
Adds a Github pull request template

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!--
```
Comment (rictic) [inline PULL_REQUEST_TEMPLATE.md:1]: I think we can make this much snappier. As is (in most of our repos) it's longer than the default size of the text field, which is super clunky. WDYT about this:

```markdown
 - [ ] CHANGELOG.md has been updated

<!--
  If this change has a user visible change (including
  bug fixes, new features, etc) please describe the change in
  CHANGELOG.md.  Consider also updating the README.
 -->
```

Comment (FredKSchott) [reply PULL_REQUEST_TEMPLATE.md:1]: 💯 💯 💯  i love it

**Comment by FredKSchott** [comment]
@rictic PTAL

**Review by rictic — APPROVED** [review]

**Comment by FredKSchott** [comment]
@rictic thanks for writing that up

**Pull request merged**